src/components/select-field/SelectField.scss

```diff
@@ -3,6 +3,8 @@
 .select-field {
     .overlay {
         min-width: 100%;
+        min-height: 34px;
```

Comment:
What is the significance of the magic number of `34`? Is it meant to display a minimum number of lines of text?